src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt

```diff
@@ -68,57 +69,15 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
         Assert.assertEquals(setOf<UUID>(), entitySetsApi.getEntitySet(linkingEs.id).linkedEntitySets)
     }
 
-    @Test
-    fun testChecksOnAddAndRemoveLinkedEntitySets() {
-        // entity set is not linking
```

Comment:
Did this group of tests move elsewhere?

Reply:
Yep, it was also present in EntitySetsTest